vite.config.js

```diff
@@ -2,21 +2,9 @@ import { defineConfig, loadEnv } from "vite";
 import react from "@vitejs/plugin-react";
 import { VitePWA } from "vite-plugin-pwa";
 
-const env = loadEnv("all", process.cwd());
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  server: {
-    host: true,
-    port: 5173,
```

Comment:
Docker needs this to be enabled, otherwise it won't expose the ports properly
The proxy argument can be removed safely, however.

Reply:
I hope it's fixed